packages/moonstone/styles/colors-light.less

```diff
@@ -199,8 +199,8 @@
 
 // RadioItem
 // ---------------------------------------
-@moon-radio-item-indicator-color: @moon-dark-gray;
-@moon-radio-item-indicator-bg-color: @moon-white;
+@moon-radio-item-indicator-color: #333333;
+@moon-radio-item-indicator-bg-color: fade(#888888, 50%);
 @moon-radio-item-focus-indicator-color: @moon-spotlight-text-color;
 @moon-radio-item-focus-indicator-bg-color: fade(#a6a6a6, 50%);
```

Comment:
The specs seems to use the same bg color for focus as well so I think we should update this to `#888888` as well.

Reply:
You meant `@moon-radio-item-focus-indicator-bg-color: fade(#888888, 50%);`  ?

Reply:
Yes

Reply:
Done!